_rules/page-has-title-2779a5.md

```diff
@@ -30,6 +30,8 @@ The rule applies to any page where the [document element](https://www.w3.org/TR/
 
 The [document element](https://www.w3.org/TR/dom/#document-element) has at least one [descendant](https://www.w3.org/TR/dom41/#concept-tree-descendant) that is an HTML `title` element.
 
+**Note**: It is invalid HTML to put the `title` inside the `body` of a page. This does not cause an accessibility issue, as browsers will still pick up a `title` element in the `body` if it is the first element on the page.
```

Comment:
Suggested rewording to get both points from the original issue https://github.com/act-rules/act-rules.github.io/issues/735 covered:

> **Note:** According to the [HTML specification for the `title` element](https://html.spec.whatwg.org/multipage/semantics.html#the-title-element), `title` elements can only be [descendants](https://www.w3.org/TR/dom41/#concept-tree-descendant) of the `head` element, and there can only be one `title` element. However, browsers will pick up the first `title` element that is a [descendant]() of the [document element](https://www.w3.org/TR/dom/#document-element) and use that as the [document title](https://html.spec.whatwg.org/multipage/semantics.html#the-title-element), also if the `title` is not a [descendant]() of the `head` element, or if there are multiple `title` elements. So even though it is invalid HTML, having `title` elements in the `body` or other places, or having multiple `title` elements don't cause any accessibility issues and is therefore not included as failures for this rule.